addons/assigngear/script_component.hpp

```diff
@@ -40,6 +40,50 @@
 
 /* assignGear specific macros */
 #define GETGEAR(var) [_cfg >> var] call CFUNC(getCfgEntryFromPath)
+
+/* assignGear _loadout indexes */
+// Appearance
+#define IDX_DISPLAY_NAME 0
+#define IDX_FACES 1
+#define IDX_INSIGNIAS 2
```

Comment:
Need to careful as the index here is also the order it is applied(!). With respect as this PR is tweaking weapon/mag stuff it should only be re-ordering relevant things.

In this case insignias should be applied after the uniform has been dealt with, as I believe now that you have tweaked it to do in the insignia first it won't be applied properly and thus breaks insignias. As insignias are just textures set on the uniform.

Reply:
Backpack items was also previously before items (!index wise), and as these will be executed in index order, this PR does tweak the order.

Reply:
Okay, will take a look, are there any other gotcha's with the ordering? Going to add some fat comments about it.

Reply:
Those are the ones I can think of.

Reply:
> With respect as this PR is tweaking weapon/mag stuff it should only be re-ordering relevant things.

Makes sense.